cmd/crank/beta/validate/cmd.go

```diff
@@ -80,6 +82,9 @@ Examples:
   # Validate all resources in the resourceDir folder against the extensions in the extensionsDir folder using provided
   # cache directory and clean the cache directory before downloading schemas
   crossplane beta validate extensionsDir/ resourceDir/ --cache-dir .cache --clean-cache
+
+  # Validate resources with old resources
```

Comment:
A bit more context would be helpful here. Why would one want to have old resources?

Reply:
The old resources feature is useful for developers who want to validate a CEL rule that requires oldSelf. For example, a CEL rule for immutable is set something like `oldSelf == self`. It needs to have old resource in place to complete a validation. So, in short, enable old resources only when a CEL rule needs `oldSelf` to calculate the validation result.

I think I can add a bit more context in `cmd.go` to have more clear picture on what cases are good to use `--old-resources` feature.

Reply:
Nono, I got the point and I agree it's useful, just feeling we should explain that in the help message a bit more in detail

Reply:
Yes, agree, I will add more context in help message. 😄

Reply:
@phisco Done, let me know what you think!